doc/spec/estimation/forest.rst

```diff
@@ -1,10 +1,156 @@
 Forest Based Estimators
 =======================
 
-\ 
```

Comment:
As I recall this line is necessary because of how sphinx identifies page headings.  At the top of a document, if you have a heading followed by regular whitespace and then a subheading then the first heading is assumed to be the title and the subheading is assumed to be a subtitle, rather than a document subheading.  This can be avoided by incorporating an escaped space (or any actual text) in between the document title and the subheading.

Reply:
So what do I need to add here?

Reply:
Just don't delete line 4.

Reply:
Done